src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/AsyncCompletionLogger.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Internal.Log;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal class AsyncCompletionLogger
+    {
+        private static readonly LogAggregator s_logAggregator = new LogAggregator();
+
+        internal enum ActionInfo
+        {
+            SessionHasExperimentEnabled,
+
+            // TargetTypeFilterChosenInSession / SessionContainsTargetTypeFilter indicates % of the time 
+            // the Target Type Completion Filter is chosen of the sessions offering it.
+            SessionContainsTargetTypeFilter,
+            TargetTypeFilterChosenInSession,
+
+            // CommitItemWithTargetTypeFilter / CommitWithTargetTypeCompletionExperimentEnabled indicates 
+            // % of the time a completion item is committed that could have been picked via the Target Type 
+            // Completion Filter.
+            CommitWithTargetTypeCompletionExperimentEnabled,
+            CommitItemWithTargetTypeFilter,
+        }
+
+        internal static void LogCommitWithTargetTypeCompletionExperimentEnabled() =>
```

Comment:
nit: I think a single `void IncreaseCount(ActionInfo kind)` would be sufficient, having one method for each kind doesn't make it easier to call/read

Reply:
FWIW, the difference at callsites is:
```C#
// Before:
                AsyncCompletionLogger.LogCommitWithTargetTypeCompletionExperimentEnabled();
// After:
                AsyncCompletionLogger.IncreaseCount(AsyncCompletionLogger.ActionInfo.CommitWithTargetTypeCompletionExperimentEnabled);
```